### ISSUE TITLE
Hypothesis: too heavy file

### ISSUE THREAD

**Issue by AtomicNess123** [karlicoss/orger#24]
Hi, I have been using orger to for my hypothesis database. However, the org file has become too big at this point.  

Do you know of a way to start syncing into a second file after the file has become too big? (or start syncing into a second file from a predefined time point instead of syncing into the same file)? Thanks!

**Comment by karlicoss** [comment]
Hi! Yeah, it's soon going to be a problem for me as well. Not supported as is at the moment, but I might look at it soon-ish!

**Comment by hwiorn** [comment]
@karlicoss 
I have 1k+ annotations. It's not many.
But it seems to be quite slowly to load it at first when I'm trying to link it to org-roam notes.
 
How about these Orger options?
1. Split org files by numbers(e.g: 5000 annotations per file).
  - Output: HypView-0001.org, HypView-0002.org, ...
2. Split org files per on web page. 
  - Output: HypView-github-karlicoss-orger-issues-24.org 
3. Split org files per on hypothesis annotation group.
  - Output: HypView-PromnesiaDev.org

1 or 2 can be implemented inside Orger itself. 3 should change the hypexport to handle hypothesis group fields for Orger.

**Comment by AtomicNess123** [comment]
Yes, I have +10k annotations, and I can't use the file in Agenda anymore 😆

**Comment by karlicoss** [comment]
Yeah, @hwiorn I think 1 makes the most sense -- in addition to it being possible to implement in Orger itself, it's also can be completely agnostic of Hypothesis, so it'll also work for other modules. Perhaps it could be a code change somewhere around this line https://github.com/karlicoss/orger/blob/1a511810f193b69a427aa88c51612fadd46b66f1/src/orger/org_view.py#L131

Another option would be to split by file size, although I'd imagine with something like 1000 annotations per file it won't matter.

**Comment by iburunat** [comment]
Hi! Similar issue here. Has this been implemented?
